cmd/argocd/commands/applicationset_test.go

```diff
@@ -185,7 +185,7 @@ SyncPolicy:         Automated
 			printAppSetSummaryTable(tt.appSet)
 			w.Close()
 
-			out, err := ioutil.ReadAll(r)
+			out, err := io.ReadAll(r)
```

Comment:
```suggestion
			out, err := os.ReadAll(r)
```

Reply:
Let's keep things a little simpler and just use the `os` package to avoid additional imports.